src/workerd/api/actor-state.c++

```diff
@@ -937,12 +951,14 @@ kj::OneOf<jsg::Ref<DurableObjectId>, kj::StringPtr> ActorState::getId(jsg::Lock&
 DurableObjectState::DurableObjectState(jsg::Lock& js,
     Worker::Actor::Id actorId,
     jsg::JsRef<jsg::JsValue> exports,
+    jsg::JsValue props,
```

Comment:
Nit: bit inconsistent here with `exports` being passed in as `jsg::JsRef<jsg::JsValue>` and `props` as just a `jsg::JsValue` since they both get stored in `jsg::JsRef<jsg::Value>` ... likely better for `props` to be a `jsg::JsRef<jsg::JsValue>` here?

Reply:
Added a commit that makes the inverse change, so now they are both `jsg::JsValue`. I somewhat prefer this as it's cleaner to look at and `JsRef`'s move constructor is actually a bit complicated.